src/pkg/blob/models/blob.go

```diff
@@ -71,6 +72,7 @@ type Blob struct {
 	UpdateTime   time.Time `orm:"column(update_time);auto_now_add" json:"update_time"`
 	Version      int64     `orm:"column(version)" json:"version"`
 	CreationTime time.Time `orm:"column(creation_time);auto_now_add" json:"creation_time"`
+	Repositories []string  `orm:"-" json:"repositories"`
```

Comment:
Can we do it different in gc job?  

I don't think keeping this reference makes sense.

Reply:
removed.